### PR TITLE
KB-8826: Align to the PIP8 naming conventions

### PR DESCRIPTION
**Pull request by amedeepotier**
Objective of this PR is to align with the PIP8 standard. 

The only task done on this ticket is Class and Variable renaming.

### CODE REVIEW

File: openchatbotclient/client_group.py [hunk 1]
```diff
@@ -11,12 +11,13 @@
...
-from . import client, response_group
+from .client import Client
+from .response_group import ResponseGroup
 
-class client_group(list):
+class ClientGroup(list):
 
     def append(self, cli):
-        assert isinstance(cli, client)
+        assert isinstance(cli, Client)
```
Comment (andreyjamer) [inline openchatbotclient/client_group.py:20]: `cli` here means `client` or `command line interface`? If client, then may be better to name is client?

File: openchatbotclient/response.py [hunk 2]
```diff
@@ -69,29 +69,21 @@
...
-class response:
-
-    def __init__(self, cli: client, data: dict):
+    def __init__(self, client, data: dict):
         """client is a client instance
            data is a json
         """
-        self.cli = cli
+        self.client = client
         self.jso = data
```
Comment (andreyjamer) [inline openchatbotclient/response.py:79]: Can `jso` be renamed into `json`?

**Review by andreyjamer — CHANGES_REQUESTED** [review]

File: openchatbotclient/client_group.py (hunk 1 above)
Comment (amedeepotier) [reply openchatbotclient/client_group.py]: yes, it means "client". I renamed the variable accordingly

File: openchatbotclient/response.py (hunk 2 above)
Comment (amedeepotier) [reply openchatbotclient/response.py]: yes, jso renamed to json

File: openchatbotclient/client_group.py [hunk 3]
```diff
@@ -27,7 +28,7 @@ def ask(self, userId: str, query: str, lang: str = None, location: str = None, m
            Returns an instance of response_group
         """
 
-        json_result_list = response_group.response_group()
+        json_result_list = ResponseGroup()
 
         for cli in self:
```
Comment (andreyjamer) [inline openchatbotclient/client_group.py:33]: Here we still have `cli`

**Review by andreyjamer — CHANGES_REQUESTED** [review]

File: openchatbotclient/client_group.py (hunk 3 above)
Comment (amedeepotier) [reply openchatbotclient/client_group.py]: Corrected

**Review by andreyjamer — APPROVED** [review]

**Pull request merged**